egs2/owsm_ctc_v3.1/s2t1/README.md

```diff
@@ -0,0 +1,14 @@
+# OWSM-CTC v3.1
```

Comment:
It would be more useful if you provide more data format information.

Reply:
I added explanation about the data format.